lib/withHotKeys.js

```diff
@@ -40,9 +40,11 @@ import HotKeys from './HotKeys';
  * @returns {function} Returns the HOC-wrapped component.
  *
  * @param {Object} keyMap an action-to-keyboard-key mapping
+ * @param {boolean} focused Whether HotKeys should behave as if it has focus in the browser
+ * @param {Object} attach The DOM element the keyboard listeners should be attached to
  * @summary An HOC that provides the wrappedComponent with the ability to implement keyboard actions
  */
-const withHotKeys = (keyMap) => ((Component) =>
+const withHotKeys = (keyMap, focused, attach) => ((Component) =>
```

Comment:
It seems likely that there may be additional props added in the future, requiring additional arguments. We then may have issues if some of those parameters are optional - we end up having to pass a long sequence of parameters that we don't necessarily care about to get to the one that we do.

A single second argument of options will allow for expansion and only require the library user to specify the options that they care about:

```
const withHotKeys = (keyMap, options) => ((Component) =>
// ... 
```

Reply:
Agreed. Fixed!